code/modules/clothing/gloves/special.dm

```diff
@@ -49,6 +50,16 @@
 
 	QDEL_NULL(pull_component_weakref)
 
+/obj/item/clothing/gloves/cargo_gauntlet/security
+	name = "\improper B.R.I. gauntlets"
+	desc = "These heavy duty gauntlets can latch on to handcuffs making them very challenging to escape. Their bulky design might make firing guns a bit harder though."
```

Comment:
You should at least explain what "BRI" stands for

Also ignorable nit: The "though" at the end reads too casually to me, I don't think descriptions should have stuff like that